lib/rucio/common/utils.py

```diff
@@ -611,18 +611,18 @@ def default(self, obj):  # pylint: disable=E0202
         return json.JSONEncoder.default(self, obj)
 
 
-def render_json(**data):
-    """ JSON render function
+def render_json(list_: Optional[list[Any]] = None, **dict_) -> str:
+    """ JSON render function for list and dicts
```

Comment:
```suggestion
    """ Render a list or a dict as a JSON-formatted string.
```

I think the docstring should make it clearer that this function renders either a list or a dict, "for list and dicts" might indicate that you can pass multiple inputs and it would work.